package/kedro_viz/data_access/managers.py

```diff
@@ -173,9 +191,15 @@ def add_pipeline(self, registered_pipeline_id: str, pipeline: KedroPipeline):
                 self.registered_pipelines.add_node(
                     registered_pipeline_id, output_node.id
                 )
-                if isinstance(output_node, TranscodedDataNode):
-                    output_node.original_name = output
-                    output_node.original_version = self.catalog.get_dataset(output)
+
+                # For older catalog
+                if (
```

Comment:
quick question; for new catalog we don't need to do this because it is done on Kedro itself right?

Reply:
Yes exactly with DataCatalog 2.0, the built-in lazy logic means we don't have to manually store or fetch the original dataset object (original_version, original_name) in our Viz code.